src/wallet/wallet.h

```diff
@@ -262,13 +262,6 @@ inline std::optional<AddressPurpose> PurposeFromString(std::string_view s)
     return {};
 }
 
-struct CRecipient
```

Comment:
this should be its own commit, might not be needed if we instead V0SilentPaymentDestination as a CTxDestination type